frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.tsx

```diff
@@ -76,12 +76,19 @@ export const ProfileForm: React.FC<ProfilePageProps> = ({
     wouldAcceptTemporary,
     poolCandidates,
     experiences,
+    isProfileComplete,
   } = profileDataInput;
 
   const intl = useIntl();
   const paths = useApplicantProfileRoutes();
   const locale = getLocale(intl);
 
+  if (isProfileComplete) {
+    sessionStorage.setItem("preProfileStatus", "Complete");
```

Comment:
Some concerns as other pages - use helper for storage and constrain strings.

Reply:
One of the things the helpers do is serialize and deserialize the stored values, which allows you to store any kind of value, not just strings. This would let you store the boolean value directly: 
`setInSessionStorage("isProfileComplete", isProfileComplete)`

Reply:
> One of the things the helpers do is serialize and deserialize the stored values, which allows you to store any kind of value, not just strings. This would let you store the boolean value directly: `setInSessionStorage("isProfileComplete", isProfileComplete)`

Ok! Thank you!